packages/neuron-ui/src/components/PageContainer/pageContainer.module.scss

```diff
@@ -30,7 +30,7 @@
 
       .syncStatus {
         height: 32px;
-        background-color: #FFFFFF99;
+        background-color: #ffffff99;
```

Comment:
The case of the css needs to be uniform.

Reply:
We can merge this issue and discuss it together on https://github.com/nervosnetwork/neuron/pull/2645#discussion_r1184737928